code/modules/food_and_drinks/food/foods/side_dishes.dm

```diff
@@ -107,6 +107,14 @@
 	list_reagents = list("nutriment" = 5, "vitamin" = 1)
 	tastes = list("rice" = 1)
 
+/obj/item/food/snacks/boiledrice/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/stack/seaweed))
+		var/obj/item/stack/seaweed/S = I
+		new /obj/item/food/snacks/onigiri(get_turf(src))
+		S.use(1)
+		qdel(src)
+		return
```

Comment:
This is how its done with other foods that have a, use item on another to transform it, interaction? No transfer of reagents or anything?

Reply:
This could use a reagent transfer probably, otherwise you might miss out on nutriments from the rice

Reply:
latest commit should resolve, lemme know if I did it wrong